File: src/your_order_please/java/Order.java
package your_order_please.java;

import java.util.Arrays;
import java.util.Comparator;

public class Order {
    public static String order(String words) {
        if (words.equals("")) {
            return "";
        }
        String[] arr = words.split(" ");
        Arrays.sort(arr, Comparator.comparingInt(word -> Integer.parseInt(word.replaceAll("\\D+", ""))));
        return String.join(" ", arr);
    }
}

File: src/kyu_6/your_order_please/js/order.js
function order(words){
    if (words === "") {
        return "";
    }

    const str = words.split(" ");

    str.sort((a, b) => {
        const numA = parseInt(a.match(/\d+/)[0]);
        const numB = parseInt(b.match(/\d+/)[0]);
        return numA - numB;
    });

    return str.join(" ");
}

module.exports = order;